x-pack/legacy/plugins/siem/public/apps/plugin.tsx

```diff
@@ -25,7 +32,11 @@ export class Plugin {
     this.chrome = chrome;
   }
 
-  public start(core: LegacyCoreStart, plugins: PluginsStart) {
+  public start(core: StartCore, plugins: StartPlugins) {
+    // TODO(rylnd): once we're on NP, we can populate version from env.packageInfo
+    core.uiSettings.set(DEFAULT_KBN_VERSION, '8.0.0');
+    core.uiSettings.set(DEFAULT_TIMEZONE_BROWSER, 'UTC');
```

Comment:
I get an error from the service when I try to retrieve this value. I know I can get the version from the plugin's init context, but I'm unsure if/where there's an equivalent for this setting, or if the warning is just a bug.

Reply:
What error are you getting? Why do you set these settings here? Did you mean to use `core.uiSettings.get`?

Reply:
I was receiving warnings when trying to `get` these two keys. The UI Settings service did not seem to know about them and advised I set defaults. I don't see that now, though, so I'll remove these 🤷‍♂

Reply:
@joshdover Ok, the rest of my team hit this when it hit master. For a fresh app, the service doesn't know about these keys, and throws errors. On my instance, these defaults were persisted to my kibana index, which is why I didn't see the error after removing these lines.

The unknown keys here are `kbnVersion` and `timezoneBrowser`. I know that the version is available on the initializer context, so I could understand it being removed from UI settings, but I'm not sure about `timezoneBrowser`.